sln/OData.Tests.E2E.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.1000

```

Comment:
Maybe this was not meant for checkin, this kinda upgrades the solution version for everyone...

Reply:
True. Fixed.